src/index.js

```diff
@@ -1,18 +1,24 @@
+import { Ion, Viewer, createWorldTerrain, createOsmBuildings, Cartesian3, Math } from "cesium";
+import "cesium/Widgets/widgets.css";
+import "../src/css/main.css"
 
-import { Cesium3DTileset, createWorldTerrain, IonResource, Viewer } from 'cesium';
-import "cesium/Build/Cesium/Widgets/widgets.css";
-import "./css/main.css";
+// Your access token can be found at: https://cesium.com/ion/tokens.
+// This is the default access token from your ion account
+Ion.defaultAccessToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiJlYWE1OWUxNy1mMWZiLTQzYjYtYTQ0OS1kMWFjYmFkNjc5YzciLCJpZCI6NTc3MzMsImlhdCI6MTYyNzg0NTE4Mn0.XcKpgANiY19MC4bdFUXMVEBToBmqS8kuYpUlxJHYZxk';
```

Comment:
Where is this default token coming from? Are you adding the right documentation needed to cycle this?

Reply:
The default token is coming from the [CesiumJS Quickstart Guide](https://cesium.com/learn/cesiumjs-learn/cesiumjs-quickstart/). I think this is appropriate here, given that I am using the code from that guide. Please let me know if you would like me to make any edits here.